src/Pelagos/Entity/Fileset.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Pelagos\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Fileset Entity class.
+ *
+ * @ORM\Entity
+ */
+class Fileset extends Entity
+{
+    /**
+     * Total file size in the fileset.
+     *
+     * @var integer
+     *
+     * @ORM\Column(name="totalFileSize", type="integer")
+     */
+    private $totalFileSize;
+
+    /**
+     * Number of files in the fileset.
+     *
+     * @var integer
+     *
+     * @ORM\Column(name="numberOfFiles", type="integer")
+     */
+    private $numberOfFiles;
+
+    /**
+     * Set totalFileSize.
+     *
+     * @param integer $totalFileSize Total file size in bytes.
+     *
+     * @return void
+     */
+    public function setTotalFileSize(int $totalFileSize)
```

Comment:
While these properties are cool, are may we used, there is no requirement for them now. And if we are just adding them because we might use them, then you should add something like a hash too.

Another thing is that file size and number should probably be just calculated automatically, and not be able to set it, so we don't need the setters. And then, the getters are also not needed (just yet).

I don't mind if we leave this in for now, but it doesn't serve a purpose, or is needed as part as the requirements.

Reply:
Yeah I agree that these attributes are not part of the requirements, but just having this entity solely to have a relationship with Files entity, we can just have Files entity instead

Reply:
It's for all the things we -don't- know! ;-)